Classes/Common/Helper.php

```diff
@@ -393,6 +393,59 @@ public static function getIndexNameFromUid($uid, $table, $pid = -1)
         }
     }
 
+    /**
+     * Get the "label" for an UID
+     *
+     * @access public
+     *
+     * @param int $uid: The UID of the record
+     * @param string $table: Get the "label" from this table
+     * @param int $pid: Get the "label" from this page
+     *
+     * @return string "label" for the given UID
+     */
+    public static function getLabelFromUid($uid, $table, $pid = -1)
+    {
+        // Sanitize input.
+        $uid = max(intval($uid), 0);
+        if (
+            !$uid
+            || !in_array($table, ['tx_dlf_collections', 'tx_dlf_libraries', 'tx_dlf_metadata', 'tx_dlf_structures', 'tx_dlf_solrcores'])
+        ) {
+            self::devLog('Invalid UID "' . $uid . '" or table "' . $table . '"', DEVLOG_SEVERITY_ERROR);
```

Comment:
We removed` self:devLog()` in the meantime. Please use `self:log()`.

Reply:
Done!